pkg/local-disk-manager/filter/filter_disk.go

```diff
@@ -61,6 +66,7 @@ func (ld *LocalDiskFilter) NodeMatch(wantNode string) *LocalDiskFilter {
 	if wantNode == ld.localDisk.Spec.NodeName {
 		ld.setResult(TRUE)
 	} else {
+		ld.FailedMessage[v1alpha1.LocalDiskAssignFailReasonNodeUnMatch] = struct{}{}
```

Comment:
it is easy to see why disks are not match when this type error happen, and a lot events like this will record in a large cluster.

thus, the error type of `NodeName not match` may be better to ignore.